streams/us.m3u

```diff
@@ -109,8 +109,12 @@ https://stream.al-mahdi.tv/hls/1234.m3u8
 http://live.albavision.net:1123/live/albdreams.m3u8
 #EXTINF:-1 tvg-id="Alhurra.us",Alhurra (720p)
 https://mbn-ingest-worldsafe.akamaized.net/hls/live/2038900/MBN_Alhurra_Worldsafe_HLS/master.m3u8
+#EXTINF:-1 tvg-id="AlhurraAlt.us",Alhurra (Alternative) (720p)
```

Comment:
```suggestion
#EXTINF:-1 tvg-id="Alhurra.us",Alhurra (720p)
```

Reply:
Moved to `us_ythls.us`.

But this suggestion wouldn't work because it's the same `tvg-id` as the official stream!